tst/EnergyPlus/unit/PlantUtilities.unit.cc

```diff
@@ -84,3 +87,342 @@ TEST_F( EnergyPlusFixture, PlantUtilities_RegisterPlantCompDesignFlowTest1 )
 	EXPECT_EQ( TestFlowRate3, CompDesWaterFlow( 1 ).DesVolFlowRate );
 
 }
+
```

Comment:
I added a lot of tests here to the PlantUtilities functions.  I didn't exhaust the entire file to 100% coverage, but it is much closer now.